app/assets/stylesheets/_users.scss

```diff
@@ -0,0 +1,115 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+$light-grey: lighten(#b2beb5, 10%);
+
+form {
+  border-right: 2px solid $light-grey;
+}
+
+.row.container {
+  bottom: 0;
+  box-shadow: none;
+  left: 0;
+  margin: auto;
+  right: 0;
+  top: 0;
+  width: 60%;
+}
+
+h2 {
+  font-size: 35px;
+  text-align: center;
+}
+
+.row.entry {
+  height: 35px;
+  width: 450px;
+}
+
+.row.submit {
+  position: relative;
+  right: 48.1955px;
+  top: 20px;
+}
+
+.container.col.l4 {
+  text-align: right;
+}
+
+.container.col.l8 {
+  padding: 0;
+  text-align: center;
+    p {
+  margin-top: 40px;
+  position: relative;
+  right: 48.1955px;
+  }
+}
+
+.row.signup {
+  font-family: candara;
+  text-align: right;
+  p {
+  font-size: 1.2em;
+  margin-top: 10px;
+  text-align: right;
+  padding-right: 34px;
+  }
+}
+
+.btn.waves-effect.waves-light {
+  height: 40px;
+  vertical-align: center;
+  width: 150px;
+
+  &.blue {
+  background-color: #338574 !important;
+  text-align: center;
+  width: 200px;
+  }
+
+  &.facebook.blue {
+  background-color: #3b5998 !important;
+  width: 150px !important;
+  }
+
+  &.twitter.blue {
+  background-color: #4099ff !important;
```

Comment:
Color literals like `#4099ff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 4 spaces, but was indented 2 spaces